src/gemidao.js

```diff
@@ -16,7 +16,10 @@ const sms = (to, token) => request.post(route('/sms'))
 const call = (from, to, token) => request.post(route('/composto'))
     .set('Access-Token', token)
     .set('Accept', 'application/json')
-    .send({
+    .send(msg_data(from, to));
+
+function msg_data(from, to) {
+    return {
         numero_destino: to,
```

Comment:
Deixa as variáveis em inglês. E bina é o sistema de identificação, não sei se seria o melhor nome para deixar o from, prefiro os nomes originais `from` e `to`